stubs/pywin32/isapi/__init__.pyi

```diff
@@ -1,10 +1,10 @@
 from _typeshed import Incomplete
 
 class ISAPIError(Exception):
-    errno: Incomplete
-    strerror: Incomplete
+    errno: int
+    strerror: str | None
     funcname: Incomplete
-    def __init__(self, errno, strerror: Incomplete | None = None, funcname: Incomplete | None = None) -> None: ...
+    def __init__(self, errno: int, strerror: str | None = None, funcname: Incomplete | None = None) -> None: ...
```

Comment:
It's made to look like a `win32api.error`. From tests we can see other errors' `funcname` should be `str | None`.

```suggestion
    def __init__(self, errno: int, strerror: str | None = None, funcname: str | None = None) -> None: ...
```